### ISSUE TITLE
Trouble with incremental deploy: The "path" argument must be of type string. Received undefined

### ISSUE THREAD

**Issue by SimonThormeyer** [seed-run/serverless-seed#3]
Hey there,

we're having trouble with incremental deploys. During _Generating incremental deploy state_, there seems to be an exception with the above error message.   

What could be causing this / how can we fix this issue to finally be able to use incremental deploys? Would it make sense to share the entire build log / serverless template?
Here is an excerpt of the build log:

```
$ SLS_DEBUG=* serverless package --stage dev --package sls-package-output

150.69 s

Running "serverless" from node_modules

Packaging offer-tool-api for stage dev (eu-west-1)

Seed: Generating incremental deploy state...

Seed: There was a problem generating the incremental deploy state

    The "path" argument must be of type string. Received undefined

✔ Service packaged (146s)
```

Thanks in advance!

**Comment by jayair** [comment]
@SimonThormeyer not sure if you've messaged us on support about this yet?

**Comment by SimonThormeyer** [comment]
@jayair not yet, but can do if that's the preferred way of support? 
Cheers

**Comment by jayair** [comment]
Yeah it works better there. Unless of course you've found something specific to this plugin!

**Comment by SimonThormeyer** [comment]
Hey @jayair, I discovered that this wasn't directly related to seed but probably this plugin, since downgrading the version of this plugin to version `0.3.12` made the error disappear (which occurs during the `serverless package` command).
I guess leaving I'm the issue open since the problem wasn't really solved.
